guide/src/class.md

```diff
@@ -1180,7 +1183,7 @@ enum BadSubclass {
 
 An enum is complex if it has any non-unit (struct or tuple) variants.
 
-Currently PyO3 supports only struct variants in a complex enum. Support for unit and tuple variants is planned.
+Currently PyO3 supports only struct and tuple variants in a complex enum. Support for unit variants is planned.
```

Comment:
As per https://github.com/PyO3/pyo3/issues/3749 I don't think we intend to make unit variants work any time soon.

```suggestion
PyO3 supports only struct and tuple variants in a complex enum. Unit variants aren't supported at present (the recommendation is to use an empty tuple enum instead).
```